app/scripts/modules/azure/src/instance/details/instance.details.controller.js

```diff
@@ -345,19 +344,6 @@ module(AZURE_INSTANCE_DETAILS_INSTANCE_DETAILS_CONTROLLER, [UIROUTER_ANGULARJS,
         });
       };
 
-      this.showConsoleOutput = function () {
-        $uibModal.open({
-          templateUrl: InstanceTemplates.consoleOutputModal,
-          controller: 'ConsoleOutputCtrl as ctrl',
-          size: 'lg',
-          resolve: {
-            instance: function () {
-              return $scope.instance;
-            },
-          },
-        });
-      };
-
```

Comment:
I noticed that this function was not used anywhere within the html so I removed it. The only reference to `showConsoleOutput` is in the new `ConsoleOutput` component. If you know anyone who works with azure, feel free to tag them.